### PR TITLE
fix: fix no build event response

### PR DESCRIPTION
**Pull request by yokawara**
### Context
Events which doesn't trigger any jobs return error to github webhooks.
It would be better to return 204.

### Objective
* Return null when events starts no jobs.

### References
relates to:
* https://github.com/screwdriver-cd/screwdriver/pull/1445

### CODE REVIEW

File: lib/eventFactory.js [hunk 1]
```diff
@@ -206,7 +207,9 @@ function createBuilds(config) {
         .then((jobsToStart) => {
             // No jobs to start (eg: when jobs are disabled or startFrom is not valid)
             if (jobsToStart.length === 0) {
-                throw new Error('No jobs to start');
+                winston.info('No jobs to start.');
```
Comment (minzcmu) [inline lib/eventFactory.js]: Can we log out the `eventId` together with this line? that will be easier to track later on

**Review by minzcmu — CHANGES_REQUESTED** [review]

**Review by tkyi — APPROVED** [review]

File: lib/eventFactory.js (hunk 1 above)
Comment (yokawara) [reply lib/eventFactory.js]: @minz1027 
That makes sense, I added `eventId` in the log. thanks!

**Comment by yokawara** [comment]
Could you restart PR job? 🙏

**Comment by minzcmu** [comment]
@yokawara lol now you can restart the pr build, @tkyi implement a feature to allow PR author to restart their builds even if they don't have permissions against the repo

https://github.com/screwdriver-cd/screwdriver/issues/1246

**Comment by yokawara** [comment]
@minz1027 I thought I could, but I can't sign in with my github account, so I can't restart the build...
It seems that I need the right to access your SD cluster.

File: lib/eventFactory.js [hunk 2]
```diff
@@ -206,7 +207,9 @@ function createBuilds(config) {
         .then((jobsToStart) => {
             // No jobs to start (eg: when jobs are disabled or startFrom is not valid)
             if (jobsToStart.length === 0) {
-                throw new Error('No jobs to start');
+                winston.info(`No jobs to start in event ${eventId}.`);
```
Comment (d2lam) [inline lib/eventFactory.js:210]: So what happens if all jobs are disabled and user clicks start? Previously they will see the error message. We should have provide some feedback to users still right?

Comment (yokawara) [reply lib/eventFactory.js:210]: @d2lam You're right, thank you. Feedbacks should be provided to users as before.
So I fixed api to return message when events don't start any job.
https://github.com/screwdriver-cd/screwdriver/pull/1445/commits/d5e5c8850f143944f7a6342349d8fe63236e8786

**Review by yoshwata — APPROVED** [review]

**Comment by minzcmu** [comment]
@yokawara @yoshwata added you guys to the whitelist for sd.cd, so that you guys can actually see your builds there!

**Review by minzcmu — APPROVED** [review]

**Comment by minzcmu** [comment]
Good to go after fixing the test

File: lib/eventFactory.js (hunk 2 above)
Comment (d2lam) [reply lib/eventFactory.js:210]: Looks good 👍

**Review by d2lam — APPROVED** [review]

**Pull request merged**